modules/helfi_recommendations/helfi_recommendations.info.yml

```diff
@@ -9,5 +9,6 @@ dependencies:
   - helfi_api_base:helfi_api_base
   - readonly_field_widget:readonly_field_widget
   - drupal:field
+  - jsonapi_extras:jsonapi_extras
```

Comment:
Should have `search_api:search_api` as a dependency too. This is required at least by `Drupal\helfi_recommendations\EventSubscriber\SearchApiSubscriber::getSubscribedEvents()`.

Reply:
Good catch! Added.